README.md

```diff
@@ -467,7 +468,70 @@ function App() {
 }
 ```
 
-### Theming
+## sx prop
+
+A special `sx` prop lets you apply styles directly to an element. When `sx` prop is specified on a component, Pigment CSS will replace any `sx` prop with `className` and `style` props at build time.
+
+The `sx` prop works on any component, including HTML elements and 3rd-party custom components as long as it is JSX.
+
+### Usage
+
+```js
+<div sx={{ display: 'flex', flexDirection: 'column' }}>
+```
+
+For a React component, it must spread the `className` and `style` to the underlying DOM element, otherwise the styles won't be applied.
```

Comment:
nit: spread is just a way of passing props. I think "pass" (or a better word?) would work best here since some devs might think they're forced to spread props.

```suggestion
For a React component, it must pass the `className` and `style` props to the underlying DOM element, otherwise the styles won't be applied.
```